maptivist/src/main/webapp/script.js

```diff
@@ -221,6 +230,18 @@ function drawInfowindow(id, title, description, links, categories, position) {
     content: contentString
   });
 
+  google.maps.event.addListener(infowindow, 'domready', function() {
+    const id = document.getElementById("vote-button").value;
+    const params = new URLSearchParams();
+    params.append("id", id);
+    params.append("update", false);
+
+    fetch('/votes', {method: 'POST', body: params}).then(response => response.json()).then((vote) => {
```

Comment:
If you implement the servlet like I suggested, then just change `method: "POST"` to `"GET"` and you can drop the `params.append("update", false)` line.